Storage/Storage/OpenTabs.swift

```diff
@@ -0,0 +1,58 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+//
+
+import Foundation
+
+public protocol OpenTabs {
+    init(files: FileAccessor)
+    
+    func persist()
```

Comment:
What if this is called multiple times? What's the concurrency contract here? Do callers care if this succeeds?